src/styles/bloks/_card.scss

```diff
@@ -0,0 +1,107 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  max-width: ($card-with + 48px) * 4;
+  box-sizing: content-box;
+
+  &__card {
+    width: $card-with;
+    margin: 0 48px 46px 0;
+    padding: 32px 16px 16px;
+  }
+}
+
+.card {
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: $main-color;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    display: flex;
+    width: 100%;
+    height: 134px;
+    text-align: center;
+    align-self: center;
+    padding: 0 3px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    text-align: left;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 140%;
+    color: #616070;
+  }
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+    &-reviews {
+      position: relative;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    line-height: 18px;
+
+    &-title {
+      color: #616070;
+      font-weight: 400;
```

Comment:
It's the default value
![image](https://user-images.githubusercontent.com/91411692/222675732-2c874b59-93fa-4eae-9add-f57c0ce9ebcb.png)

![image](https://user-images.githubusercontent.com/91411692/222675648-684def54-5cf1-4c04-b707-21d0503a2dfb.png)

```suggestion
```